### DISCUSSION TITLE
Need to install LDT LIS and LVT

### DISCUSSION THREAD

**Discussion by srikar-iist** [NASA-LIS/LISF#940] [Q&A] [answered]
I am new to LISF, and Need to install LDT LIS and LVT, in my ubuntu system.
I have been trying to understand through reading the guide given in the site, but in vain.

Kindly Help in installation and on Learning the same.

Thanks in advance,

Srikar K

**Comment by bmcandr** [comment, marked as answer]
Hi @srikar-iist, sorry to hear you're having trouble building the LISF executables. Can you be more specific about what you are struggling with?

The guides describe the required software libraries under the "Building the Executable" sections. Install each of the required libraries (and any optional libraries you choose) according to the instructions on their websites (see the links in the guide). Before configuring/compiling you must set several environment variables as described in the "Build Instructions" section.
